fxgl-core/src/main/kotlin/com/almasb/fxgl/input/Triggers.kt

```diff
@@ -152,13 +152,13 @@ data class MouseTrigger
         return event.button == button && modifier.isTriggered(event)
     }
 
-    @Suppress("NON_EXHAUSTIVE_WHEN")
     override fun isReleased(event: InputEvent): Boolean {
         if (event is KeyEvent) {
-            when (event.code) {
-                KeyCode.CONTROL -> return modifier == InputModifier.CTRL
-                KeyCode.SHIFT -> return modifier == InputModifier.SHIFT
-                KeyCode.ALT -> return modifier == InputModifier.ALT
+            return when (event.code) {
+                KeyCode.CONTROL -> modifier == InputModifier.CTRL
+                KeyCode.SHIFT -> modifier == InputModifier.SHIFT
+                KeyCode.ALT -> modifier == InputModifier.ALT
+                else -> false
```

Comment:
Although this code has passed the test
but still not sure about the return value of else
@AlmasB would you mind checking this line of code?
Is it return isTriggered(event) or false?

Reply:
Originally, `else` would just not run, falling through till the end, so I think it's `return isTriggered(event)`.

Reply:
Done